src/debugger/debugAdapter.ts

```diff
@@ -34,6 +34,7 @@ export class DebugAdapter implements DebugAdapterTracker {
                     if (!message.arguments.restart) {
                         this.debugCommunicationService.handleStopEvent();
                     }
+                    break;
```

Comment:
You could also add a default case which would throw an `UnsupportedOperationException`. Up to you.